mitreattack/navlayers/exporters/matrix_gen.py

```diff
@@ -115,16 +119,30 @@ def __init__(self, source='taxii', local=None):
                     tc = Collection('https://cti-taxii.mitre.org/stix/collections/' + collection.id)
                     self.collections[collection.title.split(' ')[0].lower()] = TAXIICollectionSource(tc)
         elif source.lower() == 'local':
-            if local is not None:
+            if resource is not None:
                 hd = MemoryStore()
-                hd.load_from_file(local)
-                if 'mobile' in local.lower():
+                hd.load_from_file(resource)
+                if 'mobile' in resource.lower():
                     self.collections['mobile'] = hd
                 else:
                     self.collections['enterprise'] = hd
             else:
                 print('[MatrixGen] - "local" source specified, but path to local source not provided')
                 raise ValueError
+        elif source.lower() == 'remote':
+            if resource is not None:
```

Comment:
Unless I'm missing something, this won't warn the user if `remote` is used without `resource`. We probably want to catch that case and do something to warn the user that those options must be used together.

Reply:
Yeah, right now it just fails to load anything, silently, if that occurs. I'll put in a warning that triggers in that case.